anonymiser.tf

```diff
@@ -0,0 +1,54 @@
+locals {
+  court_document_anonymiser_lambda_name = "${local.environment}-court-document-package-anonymiser"
+  court_document_anonymiser_queue_name  = "${local.environment}-court-document-package-anonymiser"
+}
+module "court_document_package_anonymiser_lambda" {
+  source          = "git::https://github.com/nationalarchives/da-terraform-modules//lambda"
+  function_name   = local.court_document_anonymiser_lambda_name
+  handler         = "bootstrap"
+  timeout_seconds = 30
+  lambda_sqs_queue_mappings = {
+    court_document_package_anonymiser_queue = module.court_document_package_anonymiser_sqs.sqs_arn
+  }
+  policies = {
+    "${local.court_document_anonymiser_lambda_name}-policy" = templatefile("./templates/iam_policy/anonymiser_lambda_policy.json.tpl", {
+      anonymiser_test_input_queue         = module.court_document_package_anonymiser_sqs.sqs_arn
+      ingest_court_document_handler_queue = module.ingest_parsed_court_document_event_handler_sqs.sqs_arn
+      output_bucket_name                  = local.ingest_parsed_court_document_event_handler_test_bucket_name
+      account_id                          = var.account_number
+      lambda_name                         = local.court_document_anonymiser_lambda_name
+      tre_bucket_arn                      = module.tre_config.terraform_config["prod"]["s3_court_document_pack_out_arn"]
+      tre_kms_arn                         = module.tre_config.terraform_config["prod_s3_court_document_pack_out_kms_arn"]
```

Comment:
Same question as above and same question for line 39 and 50

Reply:
I'm going to leave this one on line 21. It's different to the line above because of the way TRE have done their config so there's not much point in moving it to a variable.